FE/frontend/src/index.css

```diff
@@ -1,3 +1,7 @@
+html {
+  font-size: 62.5%;
```

Comment:
This is generally not a good practice. Is there a reason why you set this value?

Reply:
The reason behind this is when we define that font-size, we can work easier with 'rems', because then 1rem=10px --> ( 10px/16px ) * 100% = 62.5%
Now, it is easier for us to calculate something, because 2 rems are 20px, 3 30px, etc...